commands.go

```diff
@@ -72,7 +72,10 @@ func usageValidate(cmd string) string {
 		"otherwise.\n" +
 		"\n" +
 		"USAGE:\n" +
-		"    " + cmd + " FILE\n"
+		"    " + cmd + " FILE\n" +
+		"\n" +
+		"OPTIONS:\n" +
+		"    --quiet     Quite mode.\n"
```

Comment:
"quiet" not "quite".

And the description doesn't convey what the flag actually does.

My suggestion:

```suggestion
		"    --quiet     Suppress non-error output.\n"
```

Reply:
> "quiet" not "quite".

oops!